src/server/admin/server/admin_test.go

```diff
@@ -230,6 +282,19 @@ func testExtractRestore(t *testing.T, testObjects bool) {
 			"",
 			false,
 		))
+
+		if testAuth {
+			_, err := c.SetACL(c.Ctx(), &auth.SetACLRequest{
+				Repo: pipeline,
+				Entries: []*auth.ACLEntry{
+					&auth.ACLEntry{Username: fmt.Sprintf("github:pipeline-user-%d", i+1), Scope: auth.Scope_READER},
+					&auth.ACLEntry{Username: fmt.Sprintf("github:pipeline-user-%d", i), Scope: auth.Scope_OWNER},
+					&auth.ACLEntry{Username: fmt.Sprintf("pipeline:%v", pipeline), Scope: auth.Scope_WRITER},
```

Comment:
Does it break the test to remove this entry? I think `CreatePipeline` is supposed to add the pipeline user—if it's not that may be a bug.

Reply:
When the pipeline is created the pipeline user is included in the ACL, but we need to include it here because `SetACL` doesn't just append entries to the ACL, it replaces the whole thing.

Reply:
Oh, right. I'm now realizing that this seems like kind of a shortcoming in the auth design, actually, in that it's a pretty easy way to shoot yourself in the foot (maybe we should enforce a R+M+W workflow by including a version token in SetACL/GetACL, and if you call SetACL without a version token you get rejected? Or require you to set the version token to -1, so at least you know you're YOLOing. I think we do something similar with the auth config, to prevent simultaneous writes).

For a test it doesn't really matter, but one alternative could be to use `SetScope`, (i.e. `SetScope("github:...", READER); SetScope("github:...", OWNER)`).